src/targets/gpu/fuse_mlir.cpp

```diff
@@ -556,6 +558,36 @@ struct find_mlir_attention_fused_ops : public find_mlir_standalone_attention_op
     }
 };
 
+struct find_pointwise_mlir
```

Comment:
Will this fusion have the same problems with input ranks or types not matching that I've been seeing in `fuse_reduce.cpp`?  It's not clear to me just what would happen, but I don't see any checks on the compatibility of the instructions being fused.

Reply:
No it wont.